android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -108,8 +120,25 @@ avifResult AvifImageToBitmap(JNIEnv* const env,
     LOGE("Failed to lock Bitmap.");
     return AVIF_RESULT_UNKNOWN_ERROR;
   }
+  avifImage* image;
+  avifImage cropped_image;
```

Comment:
IMPORTANT: We now require libavif clients to not allocate `avifImage` on the stack like this. They should call `avifImageCreateEmpty()` to allocate a new `avifImage`. This will allow libavif to add new struct members to the end of the struct when libavif starts to provide a stable ABI.

The JNI wrapper is in a gray area. It can be considered as part of libavif. However, I suggest we eliminate this potential issue and call `avifImageCreateEmpty()`.

Reply:
Done.